modules/cert-manager-acme-dns-providers.adoc

```diff
@@ -16,3 +16,8 @@ The {cert-manager-operator} supports the following DNS-01 providers for ACME iss
 The {cert-manager-operator} does not support using Azure Active Directory (Azure AD) pod identities to assign a managed identity to a pod.
 ====
 * Google Cloud DNS
+* Webhook
++
+cert-manager also supports out-of-tree DNS providers using an external webhook.
++
+For more information, see link:https://cert-manager.io/docs/configuration/acme/dns01/#webhook[Webhook] in the upstream cert-manager documentation.
```

Comment:
```suggestion
For more information, see link:https://cert-manager.io/docs/configuration/acme/dns01/#webhook[Webhook] in the upstream {cert-manager-operator} documentation.
```

Reply:
@snarayan-redhat @lunarwhite Same as above. 
`{cert-manager-operator}` resolves to:  **cert-manager Operator for Red Hat OpenShift**
Is using that correct in this context or should we stay with **cert-manager**? 
Here I would stay with cert-manager because I don't see a mention of Operator for Red Hat OpenShift in the linked upstream documentation. But would like your confirmation.

Reply:
I agree with this. I think I was just looking for an addition of "Operator" and not registering that the attribute has all the RH OpenShift in it too.

Reply:
"cert-manager Operator for Red Hat OpenShift"'s upstream is "cert-manager". So I suppose `cert-manager` would be OK.